.stylelintrc.cjs

```diff
@@ -43,5 +43,9 @@ module.exports = {
         // other configs ...
         'stylelint-config-prettier',
     ],
-    ignoreFiles: ['packages/*/dist/**/*.css'],
+    ignoreFiles: [
+        'packages/*/dist/**/*.css',
+        'src/components/ProgressIndicator/ProgressIndicator.scss',
+        'src/components/ProfileContent/ProfileBalance/ProfileBalance.scss',
+    ],
```

Comment:
for the time being will ignore this because of the new SASS update we pushed to master recently, and this will be removed and resolved in this card https://app.clickup.com/t/20696747/P2PS-5293